src/ansys/product/library/other_module.py

```diff
@@ -224,10 +228,10 @@ def abs(self):
         >>> my_num = library.Complex(real=1, imag=1.0)
         >>> my_num.abs
```

Comment:
@greschd I think it should print/output the result here.-It is minor and not even related to this PR. 
```suggestion
        >>> my_num.abs
        1.4142135623730951
```